planetary-social/nos
# Bugfix: NIP-05 sheet is shown after reinstalling Nos

## Issues covered
#1310 

## Description
When the user opens the app for the first time, we request the metadata to all relays but there's no way to know when that request is completed because there could be not metadata at all. Instead, what we can do, is to refrain from showing the NIP-05 sheet until we have at least some user info, that is, either the name, bio, profile, etc, are set. 

## How to test
1. Log in with an account that has a NIP-05 configured
2. Delete Nos
3. Reinstall Nos
4. Open Nos
5. The NIP-05 sheet should not be presented.
